privacyidea/lib/tokens/hotptoken.py

```diff
@@ -171,6 +201,12 @@ def get_init_detail(self, params=None, user=None):
         # If the init_details contain an OTP key the OTP key
         # should be displayed as an enrollment URL
         otpkey = self.init_details.get('otpkey')
+        # Add two-step initialization parameters to response and QR code
```

Comment:
the ``get_init_detail`` is also inherited to the TOTP token. Nice. No need to change anything for TOTP.